src/assets/scripts/airport/AirportModel.js

```diff
@@ -39,9 +41,9 @@ const DEFAULT_CTR_CEILING_FT = 10000;
 const DEFAULT_INITIAL_ALTITUDE_FT = 5000;
```

Comment:
Thought for the future (that I'd imagine you might've already had yourself):

Values like these might probably be best stored in a central configuration file, so various things can be changed globally in the system, and then multiple configuration files could be chosen from in order to implement different things. Most notably this would be with settings related to the canvas, that define the shape/size/color of data blocks, targets, position symbols, history dots, airspace boundary markings, shading, video map markings, SUA boundary markings, etc. But keeping with that theme, having some behavioral defaults such as default initial altitudes and such might fit in well with that concept.